lighthouse/tests/validator_client.rs

```diff
@@ -0,0 +1,346 @@
+use validator_client::Config;
+
+use bls::{Keypair, PublicKeyBytes};
+use serde_json::from_reader;
+use std::fs::File;
+use std::io::Write;
+use std::net::Ipv4Addr;
+use std::path::PathBuf;
+use std::process::{Command, Output};
+use std::str::from_utf8;
+use std::string::ToString;
+use tempfile::TempDir;
+
+const VALIDATOR_CMD: &str = "validator_client";
+const CONFIG_NAME: &str = "vc_dump.json";
+const DUMP_CONFIG_CMD: &str = "dump-config";
+const IMMEDIATE_SHUTDOWN_CMD: &str = "immediate-shutdown";
+
+/// Returns the `lighthouse validator_client --immediate-shutdown` command.
+fn base_cmd() -> Command {
+    let target_dir = env!("CARGO_BIN_EXE_lighthouse");
```

Comment:
The dir name is a bit misleading bc this points to a binary

```suggestion
    let lighthouse_bin = env!("CARGO_BIN_EXE_lighthouse");
```

Reply:
Yep, good call. I copied this particular piece of code from the `account_manager.rs` test file, so perhaps that should be changed too?